_rules/role-attribute-valid-value-674b10.md

```diff
@@ -86,23 +91,23 @@ Element with at least one valid `role` value.
 
 #### Failed Example 1
 
-Element with invalid `role` value.
+This [`role` attribute][role attribute] contains one token, but this token is not a valid role in any of the [WAI-ARIA specifications][].
 
 ```html
 <input role="invalid" value="123" />
 ```
 
 #### Failed Example 2
 
-Element with multiple invalid `role` value.
+This [`role` attribute][role attribute] contains two tokens, but none of these tokens is a valid role in any of the [WAI-ARIA specifications][].
 
 ```html
 <input type="text" role="invalid role" />
 ```
 
 #### Failed Example 3
 
-Element with role attribute that is not empty (""), neither a valid `role` value.
+This [`role` attribute][role attribute] contains one token, but this token is not a valid role in any of the [WAI-ARIA specifications][].
 
 ```html
 <input type="text" role="#" />
```

Comment:
I am not sure this example is really needed as it is essentially the same problem as Failed Example 1.
Should we remove it?

Reply:
Agreed, lets take it out.